horace_core/admin/cache.py

```diff
@@ -1,5 +1,4 @@
-#!/usr/bin/python
-#
+#!/usr/bin/python3.8
```

Comment:
Do not specify like this, this is not transferable. 

`#!/usr/bin/env python3`

Reply:
That's useful